tools/slang-capability-generator/capability-generator-main.cpp

```diff
@@ -755,50 +768,56 @@ SlangResult generateDefinitions(DiagnosticSink* sink, List<RefPtr<CapabilityDef>
             anyStageAtomSet.add(def->enumValue);
         }
     }
-    outputUIntSetAsBufferValues("kAnyTargetUIntSetBuffer", anyTargetUIntSetHash, anyTargetAtomSet);
-    outputUIntSetAsBufferValues("kAnyStageUIntSetBuffer", anyStageUIntSetHash, anyStageAtomSet);
-    
+    outputUIntSetGenerator("generatorOf_kAnyTargetUIntSetBuffer", anyTargetUIntSetHash, anyTargetAtomSet);
```

Comment:
Why is the buffer of elements called "generatorOf_" in the generated code? Maybe just "rawBuffer_`

Reply:
`kAnyTargetUIntSetBuffer` is the buffer, `generatorOf_kAnyTargetUIntSetBuffer` is the value generator for `kAnyTargetUIntSetBuffer`

usage is as follows:
`const static CapabilityAtomSet kAnyTargetUIntSetBuffer = generatorOf_kAnyTargetUIntSetBuffer();`